contrib_bots/lib/spotify.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+from future import standard_library
+standard_library.install_aliases()
+import json, urllib.request, urllib.parse, urllib.error
+class SpotifyHandler(object):
+    '''
+    This plugin will allow users to directly search
+    for song, artists, and albums using the Spotify API.
+    It can be called using `@spotify song/artist/album`
+    '''
+    def usage(self):
+        return '''
+            This plugin will allow users to directly search
+            for song, artists, and albums using the Spotify API
+            '''
```

Comment:
@trueskawka I've changed the description a bit if it needs to be more specific let me know.